compiler/src/test/scala/org/scalajs/nscplugin/test/JSInteropTest.scala

```diff
@@ -2115,6 +2138,11 @@ class JSInteropTest extends DirectTest with TestHelpers {
   def noJSGlobalOnMembersOfClassesAndTraits: Unit = {
 
     for (outer <- Seq("class", "trait")) {
+      // There is a bug in kindStrFor() for vars in traits in Scala 2.12+
+      val varKind =
+        if (outer == "trait" && !scala.util.Properties.versionNumberString.startsWith("2.11.")) "vals"
+        else "vars"
```

Comment:
This problem disappears in the second commit by construction, so after ~10 minutes of trying to make it work in the intermediary commit, I abandoned.

Same thing in `noJSImportOnMembersOfClassesAndTraits`.